docs/casts/seamless_nested_repos.sh

```diff
@@ -1,43 +1,43 @@
-say "DataLad provides seamless management of nested Git repositories..."
+say "DataLad makes a tree of nested Git repositories feel like a big monorepo..."
```

Comment:
Ha, first time I've seen that tagline.  I'm not sure what I think of it, though my guess is that monorepo is not really a term that'd carry much weight with most of DataLad's target audience.

Reply:
I used it, because it is used on https://www.datalad.org/for/git-users

Reply:
> I used it, because it is used on https://www.datalad.org/for/git-users

The top of `content/asciicast/seamless_nested_repos.sh` in the `datalad/datalad.org` repo tells me that that file is generated from `docs/casts/seamless_nested_repos.sh`, which I think refers to the file from this PR.  So doesn't your above quote reduce to "I used it here, because I used it here"?

Anyway, my comment was really just a comment, not an objection to using that phrasing.